tests/testinator/needs.json

```diff
@@ -0,0 +1,12 @@
+{
+	"libraries": {
+		"testinator": {
+			"repository": "git@github.com:elbeno/testinator.git",
+			"commit": "b6183c43718ccefb7896c47a1d4a9d89388890e7",
+			"project" : {
+				"header-only" : true,
+				"header-directory" : "src/include"
```

Comment:
I did think about that. I'm not sure if it would prevent the need for this, but I'll check.